src/components/Alert/Alert.module.scss

```diff
@@ -0,0 +1,59 @@
+.alert {
+  display: flex;
+  border-radius: var(--alert-border-radius);
+  background-color: var(--alert-background-color-default);
+  padding: var(--alert-padding-default);
+  color: var(--alert-font-color-default);
+  font-size: var(--alert-font-size);
+
+  &.info {
+    background-color: var(--alert-background-color-info);
+    color: var(--alert-font-color-info);
+  }
+
+  &.success {
+    background-color: var(--alert-background-color-success);
+    color: var(--alert-font-color-success);
+  }
+
+  &.warning {
+    background-color: var(--alert-background-color-warning);
+    color: var(--alert-font-color-warning);
+  }
+
+  &.danger {
+    background-color: var(--alert-background-color-danger);
+    color: var(--alert-font-color-danger);
+  }
+
+  &.compact {
+    padding: var(--alert-padding-compact);
+  }
+
+  .type-icon {
+    margin-right: var(--size-spacing-sm);
+  }
+
+  .close-icon {
+    justify-content: flex-end;
+    margin-left: auto;
+    font-weight: 700;
+
+
+    &.clickable {
+      cursor: pointer;
+    }
+  }
+
+  .alert-message {
+    &:not(:last-child) {
+      margin-right: var(--size-spacing-sm);
```

Comment:
should we define a new alert token for this instead of using the spacing token directly?